Makefile

```diff
@@ -65,10 +65,19 @@ else
 
     ifeq ($(SYS),Darwin)
         # 10.11 El Capitan does not search for header files here by default
-        CFLAGS_c += -I/usr/local/include -mmacosx-version-min=10.8
+        CFLAGS_c += -I/usr/local/include
 
         # For re-link/deploy dynamic libraries
-        LDFLAGS_c += -headerpad_max_install_names -mmacosx-version-min=10.8
+        LDFLAGS_c += -headerpad_max_install_names
+
+        # Cross-compiling support
+        ifneq ($(DARWIN_TARGET),)
+            CFLAGS_c += -target $(DARWIN_TARGET)
+            LDFLAGS_c += -target $(DARWIN_TARGET)
+        else
+            CFLAGS_c += -mmacosx-version-min=10.8
+            LDFLAGS_c += -mmacosx-version-min=10.8
+        endif
```

Comment:
Would be great if someone who enjoys macOS development untangles this, but it works at least.